lib/core/presentation/themedata_swatch_color.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
```

Comment:
how about structure to like lib/core/presentation/const/themedata_swatch_color.dart